cws2/models/phono_system.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from cws2.models.base import (
+    TransientModel,
+    TransientModelManager,
+    OwnableModel,
+    UUIDModel,
+)
+from cws2.constants import PhonoSystemStatus, PhoneCategory
+
+
+class PhonoSystem(TransientModel, OwnableModel, UUIDModel):
+    is_human = models.CharField(
+        db_index=True,
+        verbose_name=_("Default human phono system"),
+        default=False,
+        help_text=_("This is the name of the phono system."),
+    )
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=64,
+        db_index=True,
+        help_text=_("This is the name of the phono system."),
+    )
+    status = models.CharField(
+        verbose_name=_("Phono system status"),
+        max_length=1,
+        choices=PhonoSystemStatus.CHOICES,
+        default=PhonoSystemStatus.DRAFT,
+        help_text=_("This is the status of the phono system."),
+    )
+    description = models.TextField(
+        verbose_name=_("Summary"),
+        blank=True,
+        help_text=_(
+            "This is a brief summary explaining the context of the phono system."
+        ),
+    )
+
+    objects = TransientModelManager()
+
+    def __str__(self):
+        return f"'{self.name}' [Phono System={self.uuid}]"
```

Comment:
A modest proposal for the default string representation. Thinking this should only be used for internal debugging or logging.